### PR TITLE
Feature/teleport scripts collisions fixes

### PR DESCRIPTION
**Pull request by Dartemiss**
What's new?
- Duplicate init scripts problem resolve for teleport
- Collider now have a function for getting the normals of the collision (thanks @mariofv)
- Change the minimum radius of the sphere collider

### CODE REVIEW

File: Engine/Component/ComponentCollider.cpp [hunk 1]
```diff
@@ -525,4 +525,36 @@ float3 ComponentCollider::GetCurrentVelocity() const
...
+				if (obA->getWorldArrayIndex() == body->getWorldArrayIndex())
+				{
+					float4 normal(pt.m_normalWorldOnB.getX(), pt.m_normalWorldOnB.getY(), pt.m_normalWorldOnB.getZ(), -pt.getDistance());
+					collisions.push_back(normal);
+				}
+				else if (obB->getWorldArrayIndex() == body->getWorldArrayIndex())
+				{
+					float4 normal(-pt.m_normalWorldOnB.getX(), -pt.m_normalWorldOnB.getY(), -pt.m_normalWorldOnB.getZ(), -pt.getDistance());
+					collisions.push_back(normal);
+				}
```
Comment (JorxSS) [inline Engine/Component/ComponentCollider.cpp]: why don't use emplace_back?

**Review by JorxSS — APPROVED** [review]
Seems good! Should be needed some testing in gameplay but in code, it's quite clear! Good job!

File: Engine/Component/ComponentCollider.cpp (hunk 1 above)
Comment (Dartemiss) [reply Engine/Component/ComponentCollider.cpp]: Copy paste from marios function xD but ill change it

Comment (mariofv) [reply Engine/Component/ComponentCollider.cpp]: 😄

**Review by mariofv — APPROVED** [review]
Good job!

**Pull request merged**